.drone.yml

```
@@ -37,7 +37,7 @@ steps:
     pull: always
     image: golang:1.15
     commands:
-      - make golangci-lint vet
+      - make golangci-lint
```

```suggestion
      - make golangci-lint
      - go vet ./....
```

Or split make target "vet" ...

---

It's unnecessary, we have did that on before step.

---

but this should -go vet- the windows code too ...

---

It's unnecessary. We have done that on non windows lint.